modules/addressing-cuda-oom-errors.adoc

```diff
@@ -0,0 +1,51 @@
+:_module-type: PROCEDURE
+
+[id="addressing-cuda-oom-errors-for-the-single-model-serving-platform_{context}"]
+= Addressing CUDA Out of Memory errors
+
+[role="_abstract"]
+
+In certain cases, depending on the model and hardware accelerator used, the TGIS memory auto tuning algorithm may underestimate the amount of GPU memory used to process long sequences. This can lead to CUDA OOM error responses from the model server. In such cases, we recommend updating or adding additional parameters in the TGIS model-serving runtime as described in the following procedure.
+
+
+.Prerequisites
+* You have logged in to {productname-long}.
+ifdef::upstream[]
+* If you are using specialized {productname-short} groups, you are part of the admin group (for example, {odh-admin-group}) in {openshift-platform}.
+endif::[]
+ifndef::upstream[]
+* If you are using specialized {productname-short} groups, you are part of the admin group (for example, {oai-admin-group}) in {openshift-platform}.
+endif::[]
+* You have installed KServe.
```

Comment:
Should this prereq say "You have installed the single-model serving platform" and point to https://access.redhat.com/documentation/en-us/red_hat_openshift_ai_self-managed/2-latest/html/serving_models/serving-large-models_serving-large-models#about-the-single-model-serving-platform_serving-large-models?